.github/workflows/coding-standards.yml

```diff
@@ -0,0 +1,63 @@
+on:
+  pull_request: ~
+  push:
+    branches:
+      - "main"
+      - "[0-9].*"
+
+name: Coding Standards
+
+jobs:
+  php-cs-fixer:
+    name: php-cs-fixer
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php-version:
+          - 7.1
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          coverage: none
+          extensions: intl
+          php-version: ${{ matrix.php-version }}
+
+      - name: Determine composer cache directory
+        id: composer-cache
+        run: echo "::set-output name=directory::$(composer config cache-dir)"
+
+      - name: Cache dependencies installed with composer
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.directory }}
+          key: composer-${{ runner.os }}-${{ matrix.php-version }}-${{ hashFiles('composer.*') }}
+          restore-keys: |
+            composer-${{ runner.os }}-${{ matrix.php-version }}-
+            composer-${{ runner.os }}-
+            composer-
+
+      - name: Download dependencies
+        run: |
+          composer update --no-interaction --no-progress --optimize-autoloader
+          composer bin php-cs-fixer install --no-interaction --no-progress --optimize-autoloader
+
+      - name: Cache cache file for php-cs-fixer
+        uses: actions/cache@v2
+        with:
+          path: .php_cs.cache
+          key: composer-${{ runner.os }}-${{ matrix.php-version }}-${{ hashFiles('composer.*') }}
```

Comment:
`composer.lock` will be all new after each composer update, so this cache might not work as expected.

Reply:
And if we consider `vendor-bin` this should be `hashFiles('**/composer.*')`